appveyor.yml

```diff
@@ -6,6 +6,10 @@ version: 1.0.{build}-{branch}
 # We're using Visual Studio 2017
 image: Visual Studio 2017
 
+# Temporary work around for https://appveyor.statuspage.io/incidents/m2vdvw39kdk8
```

Comment:
No, I've just put this change in, to get appveyor to work.
(OK, it'd probably be better in its own PR...)